controllers/sriovnetwork_controller.go

```diff
@@ -173,8 +182,31 @@ func (r *SriovNetworkReconciler) Reconcile(ctx context.Context, req ctrl.Request
 
 // SetupWithManager sets up the controller with the Manager.
 func (r *SriovNetworkReconciler) SetupWithManager(mgr ctrl.Manager) error {
+	// Reconcile when the target namespace is created after the SriovNetwork object.
```

Comment:
please also CC me next time on the upstream discussion.

Reply:
Fine. Do you mean for every change in the operator? Or for controllers one? 
Most of the discussions are held in [upstream meetings](https://www.google.com/url?q=https://docs.google.com/document/d/1sJQMHbxZdeYJPgAWK1aSt6yzZ4K_8es7woVIrwinVwI/edit?usp%3Dsharing&sa=D&source=calendar&ust=1695915143499533&usg=AOvVaw21RJVl0ayZ4YWK60F4N4G7) on Monday and Tuesday

Reply:
Every change